### DISCUSSION TITLE
Only LSIO not BINHEX?

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#100] [Q&A] [answered]
i am using the binhex container and I followed the instructions added the 2 new volumes and put the script in the folder per the instructions (lidarr) but nothing seems to be happening, not creating an extended.conf, I also do not receive errors, my lidarr is working fine but no extended capability.

Do I really need to switch over to LSIO?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Yarp
